client/python/pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.2.1"
 description = "Armada gRPC API python client"
 readme = "README.md"
 requires-python = ">=3.7,<3.9"
-dependencies = ["grpcio~=1.46.3", "grpcio-tools~=1.46.3", "mypy-protobuf>=3.2.0"]
```

Comment:
On second thought, I think you should revert this change.  

I wasn't really saying that we have to update GRPC but that we were locked into the old version because we couldn't update.  

Talking with others, I think we should just keep the version set to the lowest we test with (1.46.3) so >=1.46.3.

Reply:
Yeah, I agree with this. If we are too strict with versions (e.g. ~= or == instead of <= or >=), it'll be difficult to co-install this.